src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
It would be better to not create it static. 
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```